Packs/BMCDiscovery/README.md

```diff
@@ -0,0 +1,2 @@
+BMC Discovery is a SaaS-based, cloud-native discovery and dependency modeling system that provides instant visibility into hardware, software, and service dependencies across multi-cloud, hybrid, and on-premises environments.

```

Comment:
* Use line 1 as the pack description.
* Move line 2 to the integration README.md file.